DocSum/ui/gradio/docsum_ui_gradio.py

```diff
@@ -152,6 +189,7 @@ def generate_summary(self, doc_content, document_type="text"):
 
         except requests.exceptions.RequestException as e:
             logger.error("Request exception: %s", e)
+            print("")
```

Comment:
why print empty string?

Reply:
Please make sure CI passes and delete useless code.

Reply:
fixed